src/App.tsx

```diff
@@ -15,36 +16,90 @@
   'Garlic',
 ];
 
+const SORT_FILD_ALPHABETICCALY = 'alphabeticcaly';
+const SORT_FILD_LENGTH = 'length';
+
+function getPreparedGoods(goods: string[], sortField: string, rever: boolean) {
+  let newVisibleGoods = [...goods];
+
+  if (sortField === SORT_FILD_ALPHABETICCALY) {
+    newVisibleGoods.sort();
+  }
+
+  if (sortField === SORT_FILD_LENGTH) {
+    newVisibleGoods.sort((good1, good2) => good1.length - good2.length);
+  }
+
+  if (rever) {
```

Comment:
The variable `rever` is used here, but it seems to be a typo. Consider renaming it to `reverse` for clarity.